libs/sdk-ui-tests-e2e/reference_workspace/lib/cleanup_tiger_workspaces.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+// (C) 2021-2022 GoodData Corporation
+
+import { cleanupExpiredTigerWorkspaces } from "@gooddata/fixtures";
+import { log } from "@gooddata/fixtures/logger.js";
+
+import {
+    E2E_SDK_WORKSPACE_PREFIX,
+    E2E_SDK_CHILD_WORKSPACE_PREFIX,
+    EXPIRED_WORKSPACE_TIME,
+} from "../constant.js";
+import "../../scripts/env.js";
+
+async function cleanupTigerWorkspace(kdTigerWorkspace) {
+    const { HOST, TIGER_API_TOKEN, SDK_BACKEND } = process.env;
+    await cleanupExpiredTigerWorkspaces(
+        kdTigerWorkspace,
+        EXPIRED_WORKSPACE_TIME,
+        ["workspaces"],
+        1000,
+        TIGER_API_TOKEN,
+        HOST,
+        SDK_BACKEND,
+    );
+}
+
+async function main() {
+    try {
+        // eslint-disable-next-line no-console
+        console.log("Clean up Expired Tiger Workspaces");
```

Comment:
Should probably use only `log` like the other places in this file?

Reply:
fixed

Reply:
fixed